test/controllers/locationOfGoods/AddressControllerSpec.scala

```diff
@@ -50,6 +52,7 @@ class AddressControllerSpec extends SpecBase with AppWithDefaultMockFixtures wit
   override def guiceApplicationBuilder(): GuiceApplicationBuilder =
     super
       .guiceApplicationBuilder()
+      .overrides(bind(classOf[Navigator]).qualifiedWith(classOf[LocationOfGoods]).toInstance(fakeNavigator))
```

Comment:
Any binding can be in the controller specs for the navigator can be removed as this is done in  AppWithDefaultMockFixtures